RELEASE.md

```diff
@@ -0,0 +1,63 @@
+# Release process for a new version of the CF conventions
+Complete the following steps in the order presented.
+
+## Documents
+
+* Check that the version (e.g. `1.10`) in `cf-conventions.adoc` is correct.
```

Comment:
Ah ha! I'd forgotten about that. This will make things much easier. Am I right in think that all we need to do is change `// :final:` to `:final:` in `version.adoc`, then? and change it back after the artefacts have been created?

Reply:
Not even that should be necessary. The way this (supposedly) works is that in [the workflow](https://github.com/cf-convention/cf-conventions/blob/514c7a2900d149df915a6a5e083ed3f377e76294/.github/workflows/adoc_build.yml#L47-L50) a command line switch is added iff the build is due to a release, which automatically produces the final version.

Reply:
Hi @zklaus and @ethanrd   - sounds good, but how do does the build now that it is a release? The current procedure simply get the results of the most recent run of the guhub actions. What am I missing? Thanks,

Reply:
Hi @davidhassell - I believe a new [release](https://github.com/cf-convention/cf-conventions/releases) should cause the workflow that @zklaus mentions above to build with the `final` attribute/tag.

Reply:
Thanks, Ethan, that makes sense to me, now.  I'll update the procedure with that in mind.